examples/annotated-http-service-kotlin/src/main/kotlin/example/armeria/server/annotated/kotlin/ContextAwareService.kt

```diff
@@ -0,0 +1,57 @@
+package example.armeria.server.annotated.kotlin
+
+import com.linecorp.armeria.common.RequestContext
+import com.linecorp.armeria.server.ServiceRequestContext
+import com.linecorp.armeria.server.annotation.Get
+import com.linecorp.armeria.server.annotation.Param
+import com.linecorp.armeria.server.annotation.ProducesJson
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.withContext
+import org.slf4j.LoggerFactory
+import java.util.concurrent.Executors
+
+class ContextAwareService {
+
+    @Get("/foo")
+    @ProducesJson
+    suspend fun foo(@Param("name") name: String, @Param("id") id: Int): FooResponse {
+        log.info("Hello $name")
+        // Make sure that current thread is request context aware
+        RequestContext.current<ServiceRequestContext>()
```

Comment:
Out of curiosity: Does `ServiceRequestContext.current()` work?

Reply:
Yes.

Reply:
Then how about using `ServiceRequestContext.current()` for brevity?

Reply:
OK :)